exporter/opentelemetry-exporter-jaeger-proto-grpc/tests/test_jaeger_exporter_protobuf.py

```diff
@@ -391,6 +393,16 @@ def test_translate_to_jaeger(self):
                         v_type=model_pb2.ValueType.STRING,
                         v_str="version",
                     ),
+                    model_pb2.KeyValue(
+                        key=_SCOPE_NAME_KEY,
+                        v_type=model_pb2.ValueType.STRING,
+                        v_str="name",
+                    ),
+                    model_pb2.KeyValue(
+                        key=_SCOPE_VERSION_KEY,
+                        v_type=model_pb2.ValueType.STRING,
+                        v_str="version",
+                    ),
                 ],
```

Comment:
Should we include the `instrumentionInfo` tags as well for testing?

Reply:
Did you mean (`otel.library.name`, `otel.library.version`)? They already exist and it's a spec requirement to include them for backward-compatibility reasons in non-otlp exporters.